transifex/pull.py

```diff
@@ -218,11 +226,16 @@ def pull(repo):
 
 
 if __name__ == '__main__':
-    settings_file = join(abspath(dirname(__file__)), 'settings.yaml')
-    with open(settings_file) as f:
-        settings = yaml.load(f)
-
-    with concurrent.futures.ProcessPoolExecutor() as executor:
-        for clone_url in settings['repos']:
-            repo = Repo(clone_url)
-            executor.submit(pull, repo)
+    try:
+        clone_url = sys.argv[1]
+        repo = Repo(clone_url)
+        pull(repo)
+    except IndexError:
```

Comment:
I don't know about a stylistic choice where there are many discussion on using exceptions as flow controls. See http://softwareengineering.stackexchange.com/questions/189222/are-exceptions-as-control-flow-considered-a-serious-antipattern-if-so-why
I am on the side of we should use exceptions for real undesired cases.